sdk/go/common/testing/diagtest/sink_test.go

```diff
@@ -15,8 +15,8 @@
 package diagtest
 
 import (
-	"fmt"
```

Comment:
Could revert this file to how it was, none of these changes are needed now

Reply:
updated :)